tests/emailValidationPOM.spec.js

```diff
@@ -0,0 +1,35 @@
+import { test, expect } from '@playwright/test';
+import WelcomePage from '../src/pageObjects/welcomePage/WelcomePage';
+import RegisterPopUp from '../src/pageObjects/welcomePage/components/RegisterPopUp';
+
+test.describe('Email validation', () => {
+    
+    test.beforeEach(async ({page}) => {
+    const welcomePage = new WelcomePage(page)
+    await welcomePage.navigate()
+    await welcomePage.signUpBtn.click()
+})
+
+test('should show "Email required" when the email field is empty', async ({page}) => {
+    
+    const registerPopUp = new RegisterPopUp(page)
```

Comment:
move this to beforeEach

Reply:
it is better to create a method in welcomePage 

e.g. openSignUpPopup and return  new RegisterPopUp

Reply:
same comments for other tests